mockall_derive/src/lib.rs

```diff
@@ -516,7 +516,7 @@ fn supersuperfy(original: &Type, levels: i32) -> Type {
             Type::BareFn(bfn) => {
                 if let ReturnType::Type(_, ref mut bt) = bfn.output {
                     let new_bt = Box::new(supersuperfy(bt.as_ref(), levels));
-                    mem::replace(bt, new_bt);
+                    let _ = mem::replace(bt, new_bt);
```

Comment:
Could instead replace this with `mem::swap` if you prefer :+1: but imho thought this was a bit more explicit

Reply:
Sorry - should've seen that we could get rid of `mem::replace` entirely